assets/src/application/ui/display/TimezoneTimeInfo/style.scss

```diff
@@ -1,6 +1,15 @@
 .ee-timezone-info {
 	display: block;
-	margin: 0 var(--ee-margin-tiny);
+	margin: 0 auto;
+	position: relative;
+	[role='dialog'] {
+		position: absolute;
+		min-width: 260px;
+		left: 0;
```

Comment:
w/o that it was acting weird: 

![image](https://user-images.githubusercontent.com/1477453/79157390-275ef380-7ddd-11ea-8cc9-b1a7b45c3964.png)